src/base/bittorrent/CMakeLists.txt

```diff
@@ -0,0 +1,225 @@
+add_library(qbt_base STATIC
```

Comment:
Doesn't seem correct to add this file.

Reply:
>Doesn't seem correct to add this file.

Apparently, I accidentally copied them to this folder.